libcxx/test/std/utilities/expected/expected.expected/ctor/ctor.copy.pass.cpp

```diff
@@ -109,6 +120,13 @@ constexpr bool test() {
     assert(!e2.has_value());
   }
 
+  {
+    // https://github.com/llvm/llvm-project/issues/92676
+    std::expected<Any, int> e1;
+    auto e2 = e1;
+    assert(e2.has_value());
+  }
```

Comment:
```suggestion
  {
    // TODO(LLVM 20): Remove once we drop support for Clang 17
    #if defined(TEST_CLANG_VER) && TEST_CLANG_VER >= 1700
    // https://github.com/llvm/llvm-project/issues/92676
    std::expected<Any, int> e1;
    auto e2 = e1;
    assert(e2.has_value());
    #endif
  }
```

I don't think it's a big deal, after all this code didn't compile anyway before your fix.